pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -41,7 +41,7 @@ var (
 )
 
 func runBuildozer(args []string) {
-	const buildozer = "_bazel/bin/external/com_github_bazelbuild_buildtools/buildozer/buildozer_/buildozer"
+	var buildozer = strings.TrimSuffix(os.Args[0], "\n") + ".runfiles/../../../../../external/com_github_bazelbuild_buildtools/buildozer/buildozer_/buildozer"
```

Comment:
Why the `strings.TrimSuffix()`? Seems like it can be removed. If you were to keep it, `strings.TrimSpace()` is probably a better choice.

Reply:
Turns out there wasn't even a newline, I kept it from the previous code that did have a newline so I just removed it all together.